configure.ac

```diff
@@ -74,9 +74,15 @@ AC_CHECK_HEADER(thenonexistentheader.h, HAVE_THENONEXISTENTHEADER_H="true")
 
 # set some ld -R nonsense
 #
+AC_ARG_ENABLE(rpath,
+[  --disable-rpath         do not hardcode runtime library paths],
```

Comment:
this should use `AC_HELP_STRING`

`[AC_HELP_STRING([--disable-rpath], [do not hardcode runtime library paths])],`